containers/elixir-phoenix-postgres/.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+
+services:
+  elixir:
+    build: .
+    args:
+      # Elixir Version: 1.9, 1.10, 1.10.4, ...
+      VARIANT: 1.10
```

Comment:
You'll need quote the values of all of these (e.g. "1.10", "1.5.4", "true", "10").  A value of `true` failed to build, and I had to quote the numbers for them to process properly.